src/docs/product/integrations/source-code-mgmt/github/index.mdx

```diff
@@ -248,9 +254,16 @@ Stack trace linking takes you from a file in your Sentry stack trace to that sam
 
         - ex. For example, everything after the branch (`main`) matches the file path of `code.py` using a source code path of `https://github.com/MeredithAnya/testing/blob/main/code.py` so you don't need to set a **stack trace root** and **source code root**.
 
-     1. If the filename in your Sentry stack trace frame doesn't match the path to your source code, you will need to replace the **stack_root** part of the filename with your **source_root** to make the filename match the source code path.
+     2. If the filename in your Sentry stack trace frame doesn't match the path to your source code, you will need to replace the **stack_root** part of the filename with your **source_root** to make the filename match the source code path.
         - ex. For example, to get `src/code.py` to match `code.py` when the source code path is `https://github.com/MeredithAnya/testing/blob/main/code.py`, change the **stack trace root** to be set as `src/`, and leave **source code root** empty.
 
+#### Stacktrace modal
```

Comment:
This information is out of date in [the issue owners page](https://docs.sentry.io/product/issues/issue-owners/) (see screenshot).

In my follow up PR, I will deal with updating it.

<img width="460" alt="image" src="https://user-images.githubusercontent.com/44410/207358143-80200cfc-2b00-483c-8ac4-6eb50e77c32a.png">